packages/next/src/server-build.ts

```diff
@@ -1025,6 +1025,8 @@ export async function serverBuild({
 
       ...redirects,
 
+      ...middleware.staticRoutes,
```

Comment:
Should we add a comment here about why this needs to come before `beforeFilesRewrites`?

Reply:
Updated in https://github.com/vercel/vercel/pull/7912/commits/658bbb8187c5db20ed39549272661ab8df6bfa56